exporter/metric/option.go

```diff
@@ -25,9 +25,7 @@ import (
 	apioption "google.golang.org/api/option"
 )
 
-var (
-	userAgent = fmt.Sprintf("opentelemetry-go %s; metric-exporter %s", otel.Version(), version)
-)
+var userAgent = fmt.Sprintf("opentelemetry-go %s; metric-exporter %s", otel.Version(), Version())
```

Comment:
Unrelated to version fix, but this should probably have `google-cloud-metric-exporter` or similar instead of just `metric-exporter` to match the `google-cloud-trace-exporter` string used for tracing (see below).

@james-bebbington @aabmass do we have this documented anywhere to make sure we use consistent agent for other languages as well?

Reply:
The trace user agent is document but the metric one is not. @aabmass - can you add that to the requirements doc?

Agree that `google-cloud-metric-exporter` would make sense here